dist/tools/pyterm/pyterm

```diff
@@ -128,7 +128,7 @@ class SerCmd(cmd.Cmd):
 
         # initialize class members
         cmd.Cmd.__init__(self)
-        self.port = port
+        self.port = "/dev/tty0" #supposed to just be 'port' (no quotes)
```

Comment:
Seems unrelated.
I'm guessing your on Windows, probably using WSL2.
You may want to try to use `PORT=/dev/tty0 BOARD=rpi-pico-w make -C tests/leds flash term` for instance.
You can change the default serial port configured by the build system using `PORT=...`

Reply:
This was not supposed to be in the version that got the PR! This was a hack I had used (because I didn't know about `PORT=...`) but I had thought I had changed that back some time ago ...